libs/net/dal/Migrations/Initial/Up/PostUp/02-Ingest.sql

```diff
@@ -144,151 +147,11 @@ INSERT INTO public.ingest (
-  , 0 -- schedule_type
-  , 3 -- retry_limit
-  , 5 -- source_connection_id
-  , 6 -- destination_connection_id
-  , DEFAULT_USER_ID
-  , ''
-  , DEFAULT_USER_ID
-  , ''
-), (
-  'Meltwater'
-  , '' -- description
-  , true -- is_enabled
-  , paperId -- media_type_id
-  , (SELECT id FROM public.source WHERE code = 'MELTWATER') -- source_id
-  , 'MELTWATER' -- topic
-  , weeklyPrintId -- product_id
-  , '{ "timeZone":"Pacific Standard Time",
-      "language":"en-CA",
-      "post":false,
-      "import":false,
-      "papername":"!@PAPER=",
-      "headline":"!@HEAD=",
-      "summary":"!@ABSTRACT=",
-      "story":"!@TEXT=",
-      "author":"!@BYLINE=",
-      "date":"!@DATE=",
-      "lang":"!@LANG=",
-      "section":"!@SECTION=",
-      "id":"!@IDNUMBER=",
-      "tags":"!@LKW=",
-      "page":"!@PAGE=",
-      "item":"**START-IO-STORY**",
-      "dateFmt":"yyyyMMdd",
-      "fileFormat":"fms",
-      "filePattern":"^(.+)<date>(.+).fms$",
-      "dateOffset": -1,
-      "sources":"Vancouver Sun=SUN&The Province=PROVINCE&Times Colonist (Victoria)=TC&National Post=POST&Kelowna Daily Courier=KELOWNA&Delta Optimist=DO&North Shore News=NSN&Burnaby Now=BNOW&New West Record=NWR&Richmond News=RNEWS&Alaska Highway News=AHN&Squamish Chief=SC&Merritt Herald=MH&Tri-City News=TCN&Coast Reporter=CORE&Dawson Creek Mirror=DCMR&Kamloops This Week=KTW&Peachland View=PV&Prince George Citizen=PGC&Oliver Chronicle=APOC" }' -- configuration
-  , 0 -- schedule_type
-  , 3 -- retry_limit
-  , 5 -- source_connection_id
-  , 6 -- destination_connection_id
```

Comment:
Hardcoded primary keys that are generated result in issues more often than not.  I've replaced them with a variable below.